coverity.yaml

```diff
@@ -0,0 +1,10 @@
+capture:
+  build:
+    build-command: SKIP_BUILD_UNDERLAY_STEPS=true /packaging/build.sh
+
+commit:
+  connect:
+    auth-key-file: /auth_key.txt
+    stream: octomap_server2_github_ci
+    url: https://coverity.ssrc.fi:443/
+    on-new-cert: trust
```

Comment:
This file looks like very few details need to change per project: mainly the `stream` needs to be:
```
  stream: <repo name>_github_ci
```

A thought from me is that could there be a `coverity.yaml.template` in the container image and it gets in the `entrypoint-coverity.sh`:

- copy `coverity.yaml.template` to `coverity.yaml`
- replace `${repo_name}` from `coverity.yaml` with [GITHUB_REPOSITORY](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) (NOTE: probably the slash in `owner/reponame` needs to be transformed)

Reply:
Currently there is only that stream name which changes, but I am not sure if that is the case always. Maybe I am wiser after few more projects being scanned, but there are flags for different compilers and possible project filtering rules as well. Of course we could have that sort of generic base version and overwrite it if needed